### ISSUE TITLE
Importing exported bot config breaks the bot

### ISSUE THREAD

**Issue by toddrob99** [toddrob99/redball#6]
1. Export config from a bot
2. Import the exported json file to a bot
3. An error is thrown and the bot becomes inaccessible--must be deleted or data fixed manually via db

**Comment by toddrob99** [comment]
```
AttributeError: 'list' object has no attribute 'split'
</span> </div> % endif <% allConfig = config.get_bot_config(bot_id) %> <div id="botConfigGrid" class="configGrid layoutGrid"> % for cat in set(c['category'] for c in allConfig): % if any((x for x in allConfig if x['category']==cat)): <div class="configCategory gridItem"> <div class="gridItemContent">
/app/redball/config.py, line 73:
return [x.strip() for x in d.split(",")]
/app/redball/config.py, line 167:
"val": deserialize_key(x["val"], x["type"]),
/app/web/templates/bots.mako, line 143:
<% allConfig = config.get_bot_config(bot_id) %>
/app/web/templates/base.mako, line 141:
</div>
/usr/local/lib/python3.7/site-packages/mako/runtime.py, line 947:
callable_(context, *args, **kwargs)
/usr/local/lib/python3.7/site-packages/mako/runtime.py, line 920:
_exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
/usr/local/lib/python3.7/site-packages/mako/runtime.py, line 883:
**_kwargs_for_callable(callable_, data)
/usr/local/lib/python3.7/site-packages/mako/template.py, line 476:
return runtime._render(self, self.callable_, args, data)
/app/redball/webserver.py, line 29:
return template.render(**kwargs)
```

**Comment by toddrob99** [comment]
`game_threads_config.json`, val is stored as a string for type=list:
```
        {
            "key": "MYTEAM_BATTING_EVENTS",
            "description": "Events that will trigger comments while configured team is batting. Separate multiple events with commas. Event codes can be found at https://statsapi.mlb.com/api/v1/eventTypes. Also available: 'scoring_play'.",
            "type": "list",
            "val": "scoring_play, triple, home_run, pitching_substitution",
            "options": [],
            "subkeys": [],
            "parent_key": ""
        },
```

Default config exported from a bot, val is stored as a list:
```
        {
            "key": "MYTEAM_BATTING_EVENTS",
            "description": "Events that will trigger comments while configured team is batting. Separate multiple events with commas. Event codes can be found at https://statsapi.mlb.com/api/v1/eventTypes. Also available: 'scoring_play'.",
            "type": "list",
            "val": [
                "scoring_play",
                "triple",
                "home_run",
                "pitching_substitution",
                "walk",
                "single",
                "double",
                "strikeout"
            ],
            "options": [],
            "subkeys": [],
            "parent_key": ""
        },
```

**Comment by toddrob99** [comment]
`webserver.py` lines 838-854:
```python
                allConfig = rbConfig.get_bot_config(bot_id)
                cleanConfig = {}
                for x in allConfig:
                    if not cleanConfig.get(x["category"]):
                        cleanConfig.update({x["category"]: []})

                    cleanConfig[x["category"]].append(
                        {
                            "key": x["key"],
                            "description": x["description"],
                            "type": x["type"],
                            "val": x["val"],
                            "options": x["options"],
                            "subkeys": x["subkeys"],
                            "parent_key": x["parent_key"],
                        }
                    )
```

`val` is deserialized into a list, must be re-serialized into a comma-separated string if `type` is `list`.

`"val": x["val"] if x["type"] != "list" else ", ".join(x["val"]),`

**Comment by toddrob99** [comment]
Fixed in v0.0.4
